### DISCUSSION TITLE
Passing date on shell format

### DISCUSSION THREAD

**Discussion by JOJ0** [beetbox/beets#4756] [Q&A] [answered]
When correcting dates, what's the best format to submit. I seems like yyyy-mm-dd is not recognized:

```
beet mod added="2009-09-01 0" enforcers 3 format:flac
Modifying 6 items.
Various - Enforcers, Volume 3 - 01 - What Would We Do (1993/GB)  [RIVET 1242] f:FLAC a:23-04/MusicBrainz |Jungle, Drum And Bass
  added: 2023-04-14 09:01:13 -> 1970-01-01 01:00:00
```

I need to trick it with passing a time as well

```
beet mod added="2009-09-01 0:0:0" enforcers 3 format:flac
Modifying 6 items.
Various - Enforcers, Volume 3 - 01 - What Would We Do (1993/GB)  [RIVET 1242] f:FLAC a:23-04/MusicBrainz |Jungle, Drum And Bass
  added: 2023-04-14 09:01:13 -> 2009-09-01 00:00:00
```

What's the best way to pass a date on the shell?

**Comment by arsaboo** [comment]
You need to pass the date in epoch format. `beet modify added=1681648018 album:"Enforcers 3"` works. May be another operation to convert the date to epoch format would do the trick.

**Reply by JOJ0** [reply]
? I dont see why this is easier than passing as

"2009-09-01 0:0:0"

? :-)

**Reply by arsaboo** [reply]
Not easy for sure. I did not know we could just use `"2009-09-01 0:0:0"`...learned something new :)

**Comment by sampsyo** [comment, marked as answer]
Hi! While I know linking to the source code is an indication that things could be documented better, here we go. The dates get parsed here:
https://github.com/beetbox/beets/blob/d05c34ec4e9d8fa9e34706ac2279b7315b36e1e0/beets/library.py#L159-L160

So the format used to parse dates is actually configurable. Here's the default:
https://github.com/beetbox/beets/blob/d05c34ec4e9d8fa9e34706ac2279b7315b36e1e0/beets/config_default.yaml#L108

So if you're interested in dropping the time part, you might consider changing that config option.
